variables.tf

```diff
@@ -47,3 +47,13 @@ variable "requires_session" {
   description = "A value that indicates whether the queue supports the concept of sessions"
   default     = false
 }
+
+variable "managed_identity_object_ids" {
```

Comment:
do we need 2 ways of doing this? just one should be fine? i.e. ids vs id

Reply:
This I borrowed from the key vault code. I kinda went in blind here so I wanted to try follow an example from other platops terraform modules.

I can change it to just ids if you feel strongly about it.

Reply:
i would just do ids, this is clean slate whereas that evolved over many years